osidb/mixins.py

```diff
@@ -206,6 +206,17 @@ def __init__(self, *args, **kwargs):
         # caching the ACLs and building the group map
         self.acls_all
 
+    def save(self, *args, **kwargs):
+        if not self.acl_read and not self.acl_write:
```

Comment:
I think this assumes that there are no ACLs set when a user creates the flaw via API. **embargoed** is however a mandatory flaw parameter and it is always translated into either public or embargo ACLs.